package com.oztechan.ccc.common.error

internal class UnknownNetworkException(cause: Throwable) : Throwable(cause)
